### ISSUE TITLE
Confirm button text color cannot be modified

### ISSUE THREAD

**Issue by 343629302** [fluttercandies/flutter_wechat_assets_picker#32]
**Describe the bug (描述)**
My theme color is yellow. But the font color of the confirmation button is white. Humble. Hope to be able to customize the button font color

**Expected behavior (期望行为)**
Ability to customize button font color

**Comment by AlexVincent525** [comment]
Try `2.2.1-dev.1` .

**Comment by AlexVincent525** [comment]
Try `2.2.1-dev.2` .

**Comment by PavieOlivier** [comment]
You need to create a theme data, to change the color you need to create your own ThemeData

1. to change the preview text  color  and the dropdown menu text color
``` 
ThemeData(
textTheme: TextTheme(
     bodyText2: TextStyle(
          color: Colors.pink
       ),
   )
)
```

2. to change the color of the confirm button when confirm is enabled add a colorScheme and set the primary to the color you need. EG: we need it green
```
colorScheme: ColorScheme(
    primary: Colors.green,
   // rest is also green
    brightness: Brightness.light,
),
```

3. And lastly to change the color of the text inside the button, sadly he hard coded it to white 

![your pic](https://img.techpowerup.org/200621/screen-shot-2020-06-21-at-3-24-07-pm.png "your pic")

Please @AlexVincent525  **edit your code** by giving it theme data parameters

**Comment by AlexVincent525** [comment]
> Please @AlexVincent525 **edit your code** by giving it theme data parameters

Try `2.2.1-dev.3` which gives color scheme to customize.

**Comment by PavieOlivier** [comment]
Thank sir I will give it a try

**Comment by AlexVincent525** [comment]
User feedback with resolved.

**Comment by AlexVincent525** [comment]
Released in `v2.2.1` .
